### ISSUE TITLE
[Feature] When `waydroid prop set persist.waydroid.multi_windows` is `true`, don't render the OS loading animation full-screen.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [waydroid/waydroid#1518]
### Is your feature request related to a problem? Please describe.

On DEs where CSD can't be overridden (like KDE Plasma 6 on Wayland) even when `waydroid prop set persist.waydroid.multi_windows` is `true`, the boot animation displays full-screen. On slower devices, this could be quite annoying, due to the time it takes to complete.

### Describe the solution you'd like.

It should display in a standard window.

### Describe alternatives you've considered.

It could be hidden, but that would be problematic for slower devices, which need invocation progress feedback.

**Comment by sidevesh** [comment]
duplicate of #1182 
I agree, this needs to be addressed, especially when multi_windows is enabled,
really jarring to suddenly get a whole black screen with booting logo when you are trying to just launch an app.

**Comment by DanHolli** [comment]
I've worked around this by starting the session on login as a startup app/command then stopping it with the post session file for GDM. It makes it so it's always available with notifications via KDE Connect

**Comment by MeechaPooch** [comment]
I agree with this. I want to be able to use android apps like native apps without the splash.

**Comment by sidevesh** [comment]
I have a pull request up that resolves this https://github.com/waydroid/waydroid/pull/2250
